bin/i18n/resources/dashboard/marketing_announcements/sync_out.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../../../dashboard/config/environment'
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_out_base'
+require_relative '../marketing_announcements'
+
+module I18n
+  module Resources
+    module Dashboard
+      module MarketingAnnouncements
+        class SyncOut < I18n::Utils::SyncOutBase
+          MARKETING_ANNOUNCEMENTS_TYPE = 'marketing_announcements'.freeze
+
+          def process(language)
+            crowdin_file_path = I18nScriptUtils.locale_dir(language[:crowdin_name_s], DIR_NAME, FILE_NAME)
+            return unless File.file?(crowdin_file_path)
+
+            unless I18nScriptUtils.source_lang?(language)
+              distribute_localization(language, crowdin_file_path)
+            end
+
+            i18n_file_path = I18nScriptUtils.locale_dir(language[:locale_s], DIR_NAME, FILE_NAME)
+            I18nScriptUtils.move_file(crowdin_file_path, i18n_file_path)
+          end
+
+          private
+
+          def distribute_localization(language, file_path)
+            crowdin_translations = I18nScriptUtils.parse_file(file_path)
+
+            i18n_data = I18nScriptUtils.to_dashboard_i18n_data(language[:locale_s], MARKETING_ANNOUNCEMENTS_TYPE, crowdin_translations)
+            target_i18n_file_path = File.join(ORIGIN_I18N_DIR_PATH, "#{MARKETING_ANNOUNCEMENTS_TYPE}.#{language[:locale_s]}.json")
```

Comment:
Since we are going to move Dashboard locales to their own dirs anyway, I would recommend starting with this
```suggestion
            target_i18n_file_path = File.join(ORIGIN_I18N_DIR_PATH, MARKETING_ANNOUNCEMENTS_TYPE, "#{language[:locale_s]}.json")
```

Reply:
I tried, but didn't manage to make it work. Since there was a big lift in creating the modules and tests. I decided to not explore that update in this PR.

Reply:
I'm not sure I understand. It's just one line change in the sync-out script and in the test. We still plan to do it, it's just that it will be harder to implement later when the Dashboard locale files are already created.

Reply:
Okey, I will give it a try.

Reply:
I don't insist, we can do it latter.
I just wanted to clarify what exactly I meant.

Reply:
Not really working so I assume it requires some additional configuration of our system.
<img width="1296" alt="Screenshot 2023-10-19 at 17 28 47" src="https://github.com/code-dot-org/code-dot-org/assets/66776217/7f0680b5-5ca7-48f3-96ce-7ae20e579fe3">